cmd/finish_image_build.go

```diff
@@ -93,9 +95,14 @@ func (f *FinishImageBuild) SetFlags(flags *flag.FlagSet) {
 		"labels.")
 	flags.IntVar(&f.diskSize, "disk-size-gb", 0, "The disk size to use when creating the image in GB. Value of '0' "+
 		"indicates the default size.")
+	flags.StringVar(&f.timeout, "timeout", "60m", "Timeout value of the image build process. Must be formatted "+
```

Comment:
Maybe we can use [flags.DurationVar](https://golang.org/pkg/flag/#DurationVar)?

Reply:
Good idea, thanks! I wasn't aware of that flag type. Done now.